packages/orchestrator-ui-components/src/components/WfoProcessList/WfoProcessesList.tsx

```diff
@@ -100,18 +100,20 @@ export const WfoProcessesList = ({
         workflowName: {
             field: 'workflowName',
             name: t('workflowName'),
+            width: '20%',
             render: (value, { processId }) => (
                 <Link href={`${PATH_WORKFLOWS}/${processId}`}>{value}</Link>
             ),
         },
         lastStep: {
             field: 'lastStep',
             name: t('step'),
+            width: '15%',
         },
         lastStatus: {
             field: 'lastStatus',
             name: t('status'),
-            width: '150',
+            width: '100',
```

Comment:
just checking: no percentages? Same for processId

Reply:
Yes, absolute is needed for fix width columns